package/kernel/linux/modules/hwmon.mk

```diff
@@ -519,6 +519,20 @@ endef
 $(eval $(call KernelPackage,hwmon-tmp421))
 
 
+define KernelPackage/hwmon-tps23861
+  TITLE:=Texas Instruments TPS23861 PoE PSE
+  KCONFIG:=CONFIG_SENSORS_TPS23861
+  FILES:=$(LINUX_DIR)/drivers/hwmon/tps23861.ko
+  AUTOLOAD:=$(call AutoLoad,60,tps23861)
```

Comment:
To be honest, I have no clue that the effect of use "priority 60" here is. Can you fill me in? It seems the alternative would be to use `AutoProbe`, which defaults to a priority of 0. Does using `AutoProbe` work, or does that cause issues?

Reply:
You caught me copy-pasting.